server/schedulers/hot_region.go

```diff
@@ -170,6 +176,23 @@ func (h *balanceHotRegionsScheduler) dispatch(typ BalanceType, cluster opt.Clust
 	return nil
 }
 
+func (h *balanceHotRegionsScheduler) analyzeStoreLoad(storesStats *statistics.StoresStats) {
+	readFlowScoreInfos := NormalizeStoresStats(storesStats.GetStoresBytesReadStat())
+	writeFlowScoreInfos := NormalizeStoresStats(storesStats.GetStoresBytesWriteStat())
+	readFlowMean := MeanStoresStats(storesStats.GetStoresBytesReadStat())
+	writeFlowMean := MeanStoresStats(storesStats.GetStoresBytesWriteStat())
+
+	var weights []float64
+	means := readFlowMean + writeFlowMean
+	if means <= minFlowBytes {
```

Comment:
Is minFlowBytes suitable for the store? Is it meaningful?

Reply:
Seems to be filtering clutter？ @Luffbee @jiyingtk

Reply:
Yes.